src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -163,6 +169,39 @@ bool MissionFeasibilityChecker::checkHomePositionAltitude(dm_item_t dm_current,
 	return true;
 }
 
+bool MissionFeasibilityChecker::checkMissionItemValidity(dm_item_t dm_current, size_t nMissionItems) {
```

Comment:
I would call it it checkMissionItemValid(). I think that's a bit more intuitive for what to expect from the return bool, and it's the same style as "feasible".